opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
> If you read the Eclipse manual, it will become obvious that we don't support OPERNUM.

the *simulator* does not support OPERNUM, but so far this has never prevented the *parser* to provide proper support (at least if it was not much effort)?

> The purpose of this PR was to not die when we encountered this keyword in a very special model I have on my desk.

yeah, but adding that integer grid property is a one-liner. (as of now the parser probably still dies if it sees something like

```
COPY
"FLUXNUM" "OPERNUM" /
/
```

?)